lib/std/os.zig

```diff
@@ -6056,30 +6056,39 @@ pub fn sendto(
             .SUCCESS => return @as(usize, @intCast(rc)),
 
             .ACCES => return error.AccessDenied,
+            .ADDRINUSE => return error.AddressInUse,
+            .AFNOSUPPORT => return error.AddressFamilyNotSupported,
             .AGAIN => return error.WouldBlock,
             .ALREADY => return error.FastOpenAlreadyInProgress,
-            .BADF => unreachable, // always a race condition
+            .BADF => unreachable, // maybe a race condition. (AF_UNIX) the sent file descriptor is not open.
+            .CONNREFUSED => return error.ConnectionRefused,
```

Comment:
Can you add all currently used man pages to the commit and to your PR text on github?

Add as appended text and all potentially used ones and the context to understand the problem and make a decision.
The second question's answer depends on how many "protocol modules" exist and which Zig wants to support. It would help to clarify first, which ones are supported currently.

Reply:
> Can you add all currently used man pages to the commit and to your PR text on github?

It is already there.